Getting-Started.adoc

```diff
@@ -37,57 +37,68 @@ kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceac
 helm init --service-account=tiller
 ----
 
-NOTE: To verify that the Tiller pod has started execute the following command: `kubectl get pod --namespace kube-system` and you should see the `tiller` pod running.
+NOTE: To verify that the tiller pod has started execute the following command: `kubectl get pod --namespace kube-system` and you should see the `tiller` pod running.
 
-==== Add the Sk8s repository to your Helm environment
+==== Add the riff repository to your helm environment
 
-Run the following commands to add the `sk8s-incubator` repository:
+Run the following commands to add the repository:
 
 ----
-helm repo add sk8s-incubator http://sk8s-incubator.cfapps.io
-helm repo update
+helm repo add riffrepo https://riff_charts.storage.googleapis.com
+helm repo update riffrepo
 ----
 
-==== Install Sk8s FaaS helm chart
+==== Install the riff helm chart
 
-Install the Sk8s Helm chart using one of the following commands.
+Install the riff helm chart as follows:
 
-For a cluster that does not have RBAC enabled use:
+- `helm search riffrepo/riff` and identify the version you're looking to install (use the latest one if unsure)
+
+For an install with default configuration and no RBAC enabled use:
 
 ----
-helm install --name demo sk8s-incubator/sk8s
+helm install riffrepo/riff --name demo --version <version>
```

Comment:
don't think we need version, I've never seen anything but the latest get installed

Reply:
sure